src/net/sourceforge/kolmafia/KoLCharacter.java

```diff
@@ -192,7 +192,7 @@ public enum Gender {
   private static int currentPP, maximumPP;
 
   // Robots only
-  private static int youRobotEnergy, youRobotScraps;
+  private static long youRobotEnergy, youRobotScraps;
```

Comment:
Bit odd a change

Reply:
AdventureResult count fields are used for many things from robot energy to items to meat. Moving it to a long across the board actually feels like a good change.